src/relay/src/__flowtests__/readInlineData.js

```diff
@@ -1,94 +0,0 @@
-// @flow
```

Comment:
Why is this test not valid anymore? It should still work, no?

Reply:
Do we really need it though? We are just exporting the type from `react-relay`

Reply:
It's there to be sure it still works as expected. How do we know the types still work now?

Reply:
If they don't work, how will we fix them?

Reply:
We add the `readInlineData` wrapper back just like we did for years. 🙂